sdks/python/apache_beam/ml/gcp/recommendations_ai.py

```diff
@@ -154,13 +155,19 @@ def process(self, element):
     request = recommendationengine.CreateCatalogItemRequest(
         parent=self.parent, catalog_item=catalog_item)
 
-    try:
-      created_catalog_item = self._client.create_catalog_item(
+    @tenacity.retry(
```

Comment:
shall we keep using our own with_exponential_backoff? cc @damccorm

Reply:
Done